### PR TITLE
Update typeorm-options.interface.ts

### LINKED ISSUE #1512: "name" parameter is deprecated by typeorm documentation, but is required while registering multiple databases in nestjs project.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

deprecated parameter "name" is required for identifying datasources in project 

### Minimum reproduction code

https://github.com

### Steps to reproduce

_No response_

### Expected behavior

It shouldn't be deprecated, or new identifier should be added

### Package version

9.0.1

### NestJS version

9.0.11

### Node.js version

16.16.0

### In which operating systems have you tested?

- [X] macOS
- [X] Windows
- [ ] Linux

### Other

_No response_

### PR DESCRIPTION
**Pull request by hfhchan-plb**
Fixes #1512, #1556

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: lib/interfaces/typeorm-options.interface.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ import { ModuleMetadata, Provider, Type } from '@nestjs/common';
 import { DataSource, DataSourceOptions } from 'typeorm';
 
 export type TypeOrmModuleOptions = {
+  /**
+   * Name of connection
+   * Must match the `name` provided to `TypeOrmModule.forRootAsync` if `forRootAsync` is used.
+  name?: string;
```
Comment (kamilmysliwiec) [inline lib/interfaces/typeorm-options.interface.ts:8]: It's commented out here ☝️

Comment (micalevisk) [inline lib/interfaces/typeorm-options.interface.ts:8]:
```suggestion
  */
  name?: string;
```

I guess

**Comment by JohannesKuehnel** [comment]
@hfhchan-plb Is this not necessary anymore, or what's the reason for the close (by deletion of you fork)?

@kamilmysliwiec Should I re-create this PR or is it not needed anymore?

**Comment by hfhchan-plb** [comment]
I might have accidentally deleted this branch while closing another issue.

**Pull request closed without merging**